quickstarts/apache-hadoop/config.yml

```diff
@@ -0,0 +1,45 @@
+
+slug: apache-hadoop
+description: |
+  ## Comprehensive monitoring quickstart for Apache Hadoop
+  By integrating Apache Hadoop with New Relic, you can get comprehensive information about the workflow by visualizing and gathering metrics and logs. Apache Hadoop metrics are statistical data that Hadoop daemons offer and are used for performance adjustment, debugging, and monitoring. Numerous default metrics are accessible and are highly helpful for debugging. 
+  
+  ## Why monitor Apache Hadoop?
+  Apache Hadoop is an open-source software package which enables distributed storage and processing of massive data by leveraging a network of computers. It utilizes the MapReduce programming model-based distributed storage and processing of massive data.
+
+  By leveraging New Relic, you can gain a more in-depth understanding of Apache Hadoop performance and health. Monitoring Apache Hadoop with New Relic will help you better understand the HDFS (Hadoop Distributed File System), blocks, the system load, the data nodes, the NodeManager, and jobs.
+
+  ## What’s included in this quickstart?
+  New Relic Apache Hadoop monitoring quickstart ability to cover quality on out-of-the-box reporting.
+
+  - Jvm metrics
+  - NameNode metrics
+  - DataNode metrics
+  - Cluster metrics
+  - Queue metrics
+  - Node manager metrics
+  - Metrics system
```

Comment:
```suggestion
  - Infrastructure metrics
```

Reply:
Hi @pkosanam,
As per your suggestion, I have made changes. kindly review it.

Thank you...